src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/ParenPadCheck.java

```diff
@@ -27,9 +27,9 @@
 
 /**
  * <p>
- * Checks the policy on the padding of parentheses; i.e. whether a space is required
+ * Checks the policy on the padding of parentheses (whether a space is required
```

Comment:
How come for previous checks like `EmptyFor`, we left the `;` in but for this one we have to remove it? To me they both look like similar wording.

Reply:
We have to remove `i.e.` as it is recognized as the end of the sentence. See [Javadoc](https://checkstyle.org/apidocs/com/puppycrawl/tools/checkstyle/checks/whitespace/package-summary.html) for `ParenPadCheck`. This can be split into two sentences: 
`Checks the policy on the padding of parentheses. I.e. whether a space...`

Reply:
ok. Other checks have their sentences like:
> Checks the padding of an empty for initializer; that is whether a white

If we are keeping the other descriptions like that, it seems to me we could the same here.

Reply:
done